plugins/node/opentelemetry-instrumentation-router/src/utils.ts

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as constants from './constants';
+import * as types from './types';
+import * as Router from 'router';
+
+export const isInternal = (fn: Function) => {
+  // Note that both of those functions are sync
+  if (fn.name === 'handle' && fn.toString() === constants.ROUTER_HANDLE_FN) {
+    return true;
+  }
+  if (fn.name === 'router' && fn.toString() === constants.ROUTE_ROUTER_FN) {
+    return true;
+  }
+  return false;
+};
+
+export const getRoute = (req: Router.RoutedRequest) =>
+  req.baseUrl + (req.route?.path || '') || '/';
+
+export const renameHttpSpan = (
```

Comment:
```suggestion
export const renameHttpSpanName = (
```

Reply:
Doesn't "rename" already imply changing something's name?

Reply:
you rename http span which sounds weird, because you rename the name of that span, not the span itself

Reply:
> rename: to give (someone or something) a new name
> — https://www.merriam-webster.com/dictionary/rename

In this case that "something" is a span. Not its name. Because we're not naming the name, but a span.

I mean... I'm more than happy to change this if we get another vote. I'm currently not convinced though.

Reply:
One "changes the name" and "renames the span".

Reply:
I agree with rauno/inikem here. If I rename a thing, I am changing its name. I wouldn't say I renamed the name of something. It would be either `changeHttpSpanName` or `renameHttpSpan`.